### ISSUE TITLE
Metrics

### ISSUE THREAD

**Issue by random649** [computational-imaging/ACORN#5]
Hi, thanks for your great work!
Could you please release your code of metrics computation? Especially I want to know how many points you used to compute chamfer distance?

**Comment by davelindell** [comment]
Sorry for the delay. I pushed code to compute the Chamfer distance and IOU scores in `metrics.py`.

**Comment by gdwei** [comment]
The compute_trimesh_chamfer function in metrics.py seems to compute Chamfer-L2 instead of Chamfer-L1 which is reported in your paper. Which metric did you actually used?

**Comment by davelindell** [comment]
Okay, I was able to double check with my co-authors and seems there was some confusion and a mistake about the metrics-- Indeed our code was following the Occupancy Networks method which used Chamfer-L2, but reported Chamfer-L1 in the paper. They later fixed this in their [codebase](https://github.com/autonomousvision/occupancy_networks/commit/2fa5496d14ea2f3340865af24dbd06b988e95651). 

So in fact, the metric should be labeled Chamfer-L2. I've updated the repo with errata to clarify this.
